src/lib/updating-element.ts

```diff
@@ -317,7 +367,22 @@ export abstract class UpdatingElement extends HTMLElement {
       },
       configurable: true,
       enumerable: true
-    });
+    };
+  }
+
+  /**
+   * Returns the property options associated with the given property.
+   * These options are defined with a PropertyDeclaration via the `properties`
+   * object or the `@property` decorator and are registered in
+   * `createProperty(...)`.
+   *
+   * Note, this method should be considered "final" and not overridden. To
+   * customize the options for a given property, override `createProperty`.
+   *
+   */
+  protected static getPropertyOptions(name: PropertyKey) {
```

Comment:
Can this be private? It seems to only be accessed from other private methods. If so, rename to `_getPropertyOptions`

Reply:
It's protected so options can be used in the element lifecycle. For example, in `updated` you might want to act on a given changed property if it has a specific option.